### DISCUSSION TITLE
表格识别-表格中包含图片如何识别

### DISCUSSION THREAD

**Discussion by Maple0709** [PaddlePaddle/PaddleOCR#14112] [Q&A]
在进行表格识别的过程中，发现有的表格中有图标或者图片，这些图标或者图片占据一个单元格的情况下，表格里面的数据会丢失。怎么能够保证图片也能识别为表格的单元格

**Comment by GreatV** [comment]
训练自定义模型：
- 收集包含图片的表格数据，标注后用于重新训练 PaddleOCR 的模型。- 
- 通过增加这类样本数据，模型能更好地识别包含图片的单元格。
